libs/common/setup.go

```diff
@@ -23,9 +23,11 @@ var (
 const DevelopmentMode = "development"
 const ProductionMode = "production"
 
+var skipAuthForMethods []string
+
 // Setup loads the .env file and sets up logging
 // also sets up tokens when the service requires auth
-func Setup(serviceName, version string, auth bool) {
+func Setup(serviceName, version string, auth bool, unauthenticatedMethods *[]string) {
```

Comment:
Create overloading function in order to prevent appended `nil` passing on every call.